frontend/components/MainContent/MainContent.tsx

```diff
@@ -0,0 +1,56 @@
+import classnames from "classnames";
+import React, { ReactChild } from "react";
+
+import ExternalURLIcon from "../../../assets/images/icon-external-url-black-12x12@2x.png";
+
+interface IMainContentProps {
+  children: ReactChild;
+  /** An optional classname to pass to the main content component.
+   * This can be used to apply styles directly onto the main content div
+   */
+  className?: string;
+}
+
+const messageClassName = "sandbox-expiry-message";
+
+const SandboxExpiryMessage = (): JSX.Element => {
+  return (
+    <div className={messageClassName}>
+      <p>Your Fleet Sandbox Expires in about 20 hours.</p>
+      <a
+        href="https://fleetdm.com/docs/deploying"
+        target="_blank"
+        rel="noreferrer"
+      >
+        Learn how to renew or downgrade
+        <img
+          alt="Open external link"
+          className="icon-external"
+          src={ExternalURLIcon}
+        />
+      </a>
+    </div>
+  );
+};
+
+const baseClass = "main-content";
+
+/**
+ * A component that controls the layout and styling of the main content region
+ * of the application.
+ */
+const MainContent = ({
+  children,
+  className,
+}: IMainContentProps): JSX.Element => {
+  const classes = classnames(baseClass, className);
+
+  return (
+    <div className={classes}>
+      <SandboxExpiryMessage />
```

Comment:
Is there any check on this so that it only displays when `config.sandbox_enabled` is set?

Reply:
i have added it in one of the commits I did today https://github.com/fleetdm/fleet/pull/6830/commits/e6558e24ebb572f075a64b9ea0cddf1dbbb00627